operator/tests/e2e-with-flags/nodepools/05-assert.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+  - command: ../../../hack/get-redpanda-info.sh
```

Comment:
NIT: The number `05` should be `03` in the fole name. That would indicate the step within test case.

Reply:
fixed